features/test/apiserver.feature

```diff
@@ -0,0 +1,27 @@
+Feature: API server Test features
+  # @author kewang@redhat.com
+  @admin
+  Scenario: Quick test for step definitions
+    Given the master version >= "4.6"
+    Given I switch to cluster admin pseudo user      
+
+    # Set WriteRequestBodies profile to audit log
+    Given as admin I successfully merge patch resource "apiserver/cluster" with:
+      | {"spec": {"audit": {"profile": "WriteRequestBodies"}}} |
```

Comment:
Change api server is a destructive action in my understanding.

Reply:
Yes, this part is for testing the step definition in operator.rb, not the real OCP test case.

Reply:
Without tag destructive, the scenarios will run in parallel.
We never know how the test cases will interfere with each other if they change the cluster at the same time.

Reply:
Added destructive tag.